modules/data.atmosphere/R/extract_ERA5.R

```diff
@@ -21,85 +21,134 @@ ERA5_extract <-
   function(lat ,
            long ,
            years ,
-           vars = NULL, 
-           data.folder) {
-    
+           dbparms,
+           vars = NULL) {
+    # Distributing the job between whatever core is available. 
+     
+
     ensemblesN <- seq(1, 10)
     
+
     tryCatch({
-      # for each ensemble
-      one.year.out <- ensemblesN %>%
-        purrr::map(function(ens) {
+       #for each ensemble
+      one.year.out <- years %>%
+        purrr::map(function(year) {
+          
+        
+          tryCatch({
+            bety <- dplyr::src_postgres(dbname   = dbparms$dbname, 
+                                        host     = dbparms$host, 
+                                        user     = dbparms$user, 
+                                        password = dbparms$password)
+            
+            con <- bety$con
+          },
+          error = function(e) {
+            PEcAn.logger::logger.severe(paste0("",e))
+          }
+          )
+         
+
+          #--- lets find the big raw tile.
+          raw.tiles <- PEcAn.DB::dbfile.input.check(
+            siteid = "1000026755",
```

Comment:
In general, site ID shouldn't be hard coded. I suspect that you should be getting this info from the met registry file.